extension.js

```diff
@@ -31,11 +29,11 @@ init();
 
 function openCurrentURL() {
     if (tokenOk) {
-        if (totalCounter === 1) {
-            chrome.tabs.create({'url': pullRequestURL});
+        if (pullRequestUrls.length === 1) {
+            chrome.tabs.create({'url': pullRequestUrls[0]});
         }
         else {
-            if (totalCounter === 0 || assigneesCounter > 0) {
+            if (pullRequestUrls.length === 0 || pullRequestUrls.length !== reviewsCounter) {
```

Comment:
can you explain me the second condition?

Reply:
if the pullRequestUrl length is not equal the length of reviews, that means, that there must be assigned pull requests, but i extracted it now!